src/Components/ClcScrn.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class ClcScrn extends React.PureComponent {
+  render() {
+    const { value } = this.props;
+    return (
+      <div className="clcScrn">
+        {value}
+      </div>
+    );
+  }
+}
+
+ClcScrn.propTypes = {
+  value: PropTypes.string.isRequired,
+};
+
+export default ClcScrn;
```

Comment:
- [ ] One of the requirements is that you should use _**functional**_ components instead of _**class-based**_ components. Kindly implement this requirement for this component. 👍 😎